compiler/src/dmd/dtoh.d

```diff
@@ -778,6 +778,16 @@ public:
             }
         }
 
+        if (tf && tf.next)
```

Comment:
I'm not sure why both of these are required, my LDC I'm developing against only required `if (tf)`